### PR TITLE
Add new section to Portal docs for Sending alerts

### PR DESCRIPTION
**Pull request by dsamojlenko**
# Summary | Résumé

Adds the Sending QR Alerts section to the About the portal secction.

| English | French |
| ----- | ----- |
image | image

### CODE REVIEW

**Review by dorianjp — COMMENTED** [review]
Just the question about visibility and permissions.

File: about/templates/includes/_nav_about.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
       <li><a {% if request.resolver_match.url_name == 'one_time_keys' %}class="active"{% endif %} href="{% url 'about:one_time_keys' %}">{% trans "One-time keys" %}</a></li>
       <li><a {% if request.resolver_match.url_name == 'give_a_key' %}class="active"{% endif %} href="{% url 'about:give_a_key' %}">{% trans "Give a key" %}</a></li>
       <li><a {% if request.resolver_match.url_name == 'help_patient_enter_key' %}class="active"{% endif %} href="{% url 'about:help_patient_enter_key' %}">{% trans "Help patient enter key" %}</a></li>
+      <li><a {% if request.resolver_match.url_name == 'sending_qr_alerts' %}class="active"{% endif %} href="{% url 'about:sending_qr_alerts' %}">{% trans "Sending alerts" %}</a></li>
```
[the remark this answers is not in the document]
Comment (dsamojlenko) [reply about/templates/includes/_nav_about.html]: Oh yeah, good catch. just added a property to the user model and added the conditional.

**Comment by dsamojlenko** [comment]
Added the permission check, ready for re-review

**Review by dorianjp — APPROVED** [review]
lgtm

**Pull request merged**